package.json

```diff
@@ -18,7 +18,8 @@
     "node": ">=0.10.0"
   },
   "scripts": {
-    "test": "xo && node test.js"
+    "test": "xo && nyc node test.js",
+    "coverage": "xo && nyc --reporter=text-lcov node test.js | coveralls"
```

Comment:
Why run XO here? You're running it in `after_success` which doesn't fail the build upon exiting with non-zero. Plus, it was run earlier, in the `test` script.